stan/math/rev/mat/functor/cvodes_ode_data.hpp

```diff
@@ -70,15 +110,11 @@ class cvodes_ode_data {
                           N_Vector* yS, N_Vector* ySdot, void* user_data,
                           N_Vector tmp1, N_Vector tmp2) {
     const ode_data* explicit_ode = static_cast<const ode_data*>(user_data);
-    const std::vector<double> y_vec(NV_DATA_S(y),
-                                    NV_DATA_S(y) + explicit_ode->N_);
-    explicit_ode->rhs_sens(explicit_ode->y0_, explicit_ode->theta_, t, y_vec,
-                           yS, ySdot);
+    explicit_ode->rhs_sens(t, NV_DATA_S(y), yS, ySdot);
     return 0;
   }
 
-  static int dense_jacobian(long int N,  // NOLINT(runtime/int)
-                            realtype t, N_Vector y, N_Vector fy, DlsMat J,
+  static int dense_jacobian(realtype t, N_Vector y, N_Vector fy, SUNMatrix J,
```

Comment:
Please add some documentation. At least say this is the implementing the type of function that's `CVDlsJacFn` and this is where the doc can be found: <https://computation.llnl.gov/sites/default/files/public/cv_guide.pdf>

Reply:
I linked the cvs_guide.pdf which also covers the sensitivity RHS.

Reply:
Thanks! That helps a ton. It's still a little short on the standards of our doc, but I think this is enough for now. I wish that doc were there before -- it would have saved me a few hours while reading this code.